data/discord.json

```diff
@@ -3,23 +3,27 @@
     {
       "name": "General",
       "channelId": "709438882504245361",
+      "color": 4219864,
```

Comment:
@christopher-dG What is this value?

Reply:
They're decimal color codes, this one is Julia Blue.

Reply:
Awesome. I will definitely use this elsewhere as well. Thanks!